src/WebAPI/Program.cs

```diff
@@ -40,12 +45,24 @@ public static void Main(string[] args)
 
         builder.Logging.AddSerilog(logger);
 
+        builder.Services.AddHttpClient();
+
+        builder.Services.AddSwaggerGen(c =>
+        {
+            c.SwaggerDoc("v1", new OpenApiInfo { Title = "CacheTask", Version = "v1" });
+
+            // Include the XML comments file
+            var xmlFile = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
+            var xmlPath = Path.Combine(AppContext.BaseDirectory, xmlFile);
+            c.IncludeXmlComments(xmlPath);
+        });
+
```

Comment:
this breaks authentication, update swager definition that is in JWTAuthenticateInjection.cs